### PR TITLE
fix: complete and check missing translations

### PR DESCRIPTION
**Pull request by mikasackermn**
# Summary

Complete and check missing translations

# How did you test this change?

Run Playground and change the language

# Checklist:

- [x] I have performed a self-review of my code

### CODE REVIEW

**Comment by samobasquiat** [comment]
In `widget/embedded/src/components/SwapsGroup/SwapsGroup.tsx` I see some titles for `loadingGroups`. Also we wrote this code` onlyShowTime={group.title === 'Today'}` that I think it should change.

**Comment by samobasquiat** [comment]
Screenshot from 2023-12-03 11-31-16
Screenshot from 2023-12-03 11-31-50
Screenshot from 2023-12-03 11-32-11
Screenshot from 2023-12-03 11-32-39

File: widget/embedded/src/components/SwapsGroup/SwapsGroup.tsx [hunk 1]
```diff
@@ -17,11 +17,11 @@ export function SwapsGroup(props: PropTypes) {
 
     const loadingGroups = [
       {
-        title: 'Today',
+        title: i18n.t('Today'),
         swaps,
       },
       {
-        title: 'Last month',
+        title: i18n.t('Last month'),
```
Comment (samobasquiat) [inline widget/embedded/src/components/SwapsGroup/SwapsGroup.tsx:24]: i18n.t is here also in the jsx. line 38. do we need both of them?

Comment (mikasackermn) [reply widget/embedded/src/components/SwapsGroup/SwapsGroup.tsx:24]: fixed

**Review by samobasquiat — APPROVED** [review]

**Comment by RanGojo** [comment]
it looks good to me but i will check if after merging ci task for crowdin.

**Review by RanGojo — APPROVED** [review]

File: widget/embedded/src/utils/date.ts [hunk 2]
```diff
@@ -28,14 +29,21 @@ export const groupSwapsByDate: GroupBy = (swaps) => {
     [
       'month',
       {
-        title: 'This month',
+        title: i18n.t('Last month'),
```
Comment (samobasquiat) [inline widget/embedded/src/utils/date.ts:32]: After rebase "week" has been added.

**Review by samobasquiat — CHANGES_REQUESTED** [review]

File: widget/embedded/src/utils/date.ts [hunk 3]
```diff
@@ -28,14 +29,21 @@ export const groupSwapsByDate: GroupBy = (swaps) => {
...
       {
-        title: 'This year',
+        title: i18n.t('Last year'),
+        swaps: [],
+      },
+    ],
+    [
+      'history',
+      {
+        title: i18n.t('History'),
```
Comment (samobasquiat) [inline widget/embedded/src/utils/date.ts:46]: We don't need History anymore. we show the year.

File: widget/embedded/src/utils/date.ts [hunk 4]
```diff
@@ -28,14 +29,21 @@ export const groupSwapsByDate: GroupBy = (swaps) => {
...
-        title: 'This month',
+        title: i18n.t('Last month'),
         swaps: [],
       },
     ],
     [
       'year',
       {
-        title: 'This year',
+        title: i18n.t('Last year'),
```
Comment (samobasquiat) [inline widget/embedded/src/utils/date.ts:39]: Please replace "Last" with "This"

**Review by samobasquiat — CHANGES_REQUESTED** [review]

File: widget/embedded/src/utils/date.ts (hunk 2 above)
Comment (mikasackermn) [reply widget/embedded/src/utils/date.ts:32]: fixed

File: widget/embedded/src/utils/date.ts (hunk 4 above)
Comment (mikasackermn) [reply widget/embedded/src/utils/date.ts:39]: fixed

File: widget/embedded/src/utils/date.ts [hunk 5]
```diff
@@ -14,28 +15,28 @@ export const groupSwapsByDate: GroupBy = (swaps) => {
...
-        title: 'This week',
+        title: i18n.t('This week'),
         swaps: [],
       },
     ],
     [
       'month',
       {
-        title: 'This month',
+        title: i18n.t('Tis month'),
```
Comment (samobasquiat) [inline widget/embedded/src/utils/date.ts:32]: This!

**Review by samobasquiat — CHANGES_REQUESTED** [review]

**Review by samobasquiat — APPROVED** [review]

**Pull request merged**